quickwit/quickwit-cli/src/index.rs

```diff
@@ -261,14 +261,16 @@ impl IndexCliCommand {
 
     fn parse_clear_args(matches: &ArgMatches) -> anyhow::Result<Self> {
```

Comment:
With new API, it makes more sense to pass owned `ArgMatches` (see `remove_subcommand`) so we can use the `remove_one`, `try_remove_one` function in subcommand parsers.